app/models/apartment.rb

```diff
@@ -0,0 +1,4 @@
+class Apartment < ActiveRecord::Base
+  validates :renter, :phone1, presence: true
+  validates :phone1, format: { without: /[a-zA-Zа-яА-Я]/, message: I18n.t('.validation.apartment.phone') }
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.